Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -219,7 +219,7 @@ - (void)handleRemoteNotificationReceived:(NSNotification *)notification
     self.remoteNotificationCallbacks[notificationId] = completionHandler;
   }
 
-  [self sendEventWithName:@"remoteNotificationReceived" body:notification.userInfo];
+  [self sendEventWithName:@"remoteNotificationReceived" body:remoteNotification];
```

Comment:
👍  this is a result of a 9 month long PR with subtle refactors over long periods of time and not tested after each refactor. Thank you for catching this, I believe what you have here is correct